.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: CI
```

Comment:
Since this is copied from Guzzle, please keep the two spaces indentation for now.

Reply:
Done.